lib/improver/spotdata_new/neighbour_finding.py

```diff
@@ -173,7 +175,11 @@ def check_sites_are_within_domain(sites, site_coords, site_x_coords,
         Args:
             sites (list of dicts):
                 A list of dictionaries defining the spot sites for which
-                neighbours are to be found.
+                neighbours are to be found. e.g.:
```

Comment:
This probably needs to go in the process method too, as this is where a user actually has to pass this in.